SimulationOutputReport/measure.rb

```diff
@@ -613,17 +614,27 @@ def get_outputs(timeseries_frequency,
     # Get zone temperatures
     if include_timeseries_zone_temperatures
       zone_names = []
+      other_side_obj_names = []
       @model.getThermalZones.each do |zone|
         if zone.floorArea > 1
           zone_names << zone.name.to_s.upcase
         end
       end
+      @model.getSurfacePropertyOtherSideCoefficientss.each do |otherside_obj|
```

Comment:
For buildings that have a water heater and/or duct in one of the new MF space types, but there are no surfaces adjacent to the MF space type, the space temperatures aren't accounted for here. We need to account for them. (We don't need to worry about appliances, which are only assigned to "other" anyway.)